app/jobs/dps_export_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class DPSExportJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    dps_export =
+      DPSExport.new(VaccinationRecord.where(exported_to_dps_at: nil)).export_csv
```

Comment:
```suggestion
      DPSExport.new(VaccinationRecord.recorded.where(exported_to_dps_at: nil)).export_csv
```

I know there's a default scope on the model, but I prefer things to be explicit. When I first read this code I thought it would be sending draft records to DPS.